src/frontends/onnx/frontend/src/op/org.openvinotoolkit/prior_box.cpp

```diff
@@ -89,6 +89,13 @@ OutputVector prior_box_clustered(const Node& node) {
     attrs.variances = node.get_attribute_value<std::vector<float>>("variance", {0.1f});
     attrs.step_heights = node.get_attribute_value<float>("step_h", 0.0f);
     attrs.step_widths = node.get_attribute_value<float>("step_w", 0.0f);
+    if (attrs.step_heights == 0.0f && attrs.step_widths == 0.0f) {
+        // According to spec, if both 'step_heights' and 'step_widths' are 0, they're updated with value of 'step'
+        // TODO: consider to move this logic to core's PriorBoxClustered::visit_attributes
+        attrs.step = node.get_attribute_value<float>("step", 0.0f);
+        attrs.step_heights = attrs.step;
+        attrs.step_widths = attrs.step;
+    }
```

Comment:
I think just fetching 'step' attribute should be enough (without check if step_heights/widths is zero)
Rest should be handled by plugins https://github.com/openvinotoolkit/openvino/blob/0b75589e27c4a69a04d73a1a5ead2e429d67ecfa/src/plugins/intel_cpu/src/nodes/mkldnn_priorbox_clustered_node.cpp#L115.
Also I can see that legacy onnx frontend also doesn't compare step_heights/widths to zero https://github.com/openvinotoolkit/openvino/blob/0b75589e27c4a69a04d73a1a5ead2e429d67ecfa/tools/mo/openvino/tools/mo/front/onnx/priorbox_clustered_ext.py#L29.

Reply:
Update: I've checked 2021.3 release and found a lot of models with step="16", step_h="0", step_w="0". So it looks pretty safe just to update ONNX frontend and PriorBoxClustered::visit_attributes.
Also checked accuracy for original model under ticket, it worked fine for CPU and GPU